src/numeric/decision_rules.jl

```diff
@@ -29,7 +29,7 @@ end
 (dr::ConstantDecisionRule)(i::Int, j::Int, x::Union{AbstractVector{Float64},AbstractMatrix{Float64}}) = dr(x)
 
 
-type BiTaylorExpansion <: AbstractDecisionRule
+@compat type BiTaylorExpansion <: AbstractDecisionRule{EmptyGrid,EmptyGrid}
```

Comment:
Here let's use the new syntax, which means either using `@compat struct` if `BiTaylorExpansion` is meant to be immutable or `@compat mutable struct` if it needs to be mutable.